src/views/RequiredSignoffs/ViewSignoff/index.jsx

```diff
@@ -43,12 +43,15 @@ export default function ViewSignoff() {
   const [type, setType] = useState('channel');
   const handleTypeChange = ({ target: { value } }) => setType(value);
 
+  console.log(products);
+  console.log(setProducts);
   useEffect(() => {
     console.log('prior to getting products');
 
     (async () => {
       const [error, result] = await tryCatch(getProducts());
 
+      console.log(error);
```

Comment:
The next PR will display an error component when an error is there. We can remove this line.